modules/lib/src/main/resources/assets/js/v6/features/shared/dialogs/new-content/NewContentDialogUpload.ts

```diff
@@ -0,0 +1,109 @@
+import {ContentSummaryAndCompareStatus} from 'src/main/resources/assets/js/app/content/ContentSummaryAndCompareStatus';
```

Comment:
This file must be part of the store. It's not a component, it's just a bunch of utility functions.